langchain4j-jina-ai/pom.xml

```
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>dev.langchain4j</groupId>
+        <artifactId>langchain4j-aggregator</artifactId>
+        <version>0.30.0</version>
+    </parent>
+
+    <artifactId>langchain4j-jina-ai</artifactId>
```

nitpick:
```suggestion
    <artifactId>langchain4j-jina</artifactId>
```
Please rename it everywhere, including package name `jinaAi` -> `jina`